### ISSUE TITLE
CE SketchUp Extension for SketchUp 2022

### ISSUE THREAD

**Issue by bee-am** [highered-esricanada/CityEngine-Sketchup-Extension#1]
Wanted to find out if SketchUp 2022 will be supported any time soon.

**Comment by luubert** [comment]
Yes, you can download the latest version with SketchUp 2022 support [here](https://github.com/highered-esricanada/CityEngine-Sketchup-Extension/releases/tag/1.2).

**Comment by bee-am** [comment]
Thanks @luubert !
